scripts/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ "${UPDATE_ON_BOOT}" = false ]; then
+    echo "Update on Boot needs to be enabled for auto updating"
+    exit 1
+fi
+
+CURRENTBUILD=$(awk '/buildid/{ print $2 }' < palworld/steamapps/appmanifest_2394010.acf)
```

Comment:
Using relative instead of absolute path.
```
palworld/steamapps/appmanifest_2394010.acf: No such file or directory
```

```suggestion
CURRENTBUILD=$(awk '/buildid/{ print $2 }' < /palworld/steamapps/appmanifest_2394010.acf)
```